app/_data/tables/install_options.yml

```diff
@@ -13,50 +13,60 @@ features:
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/docker.png
```

Comment:
Is there an Alpine logo that we can use? This is the package, not the docker image